frontend/packages/ux-editor/src/components/Properties/PageConfigPanel/PageConfigPanel.tsx

```diff
@@ -28,14 +33,28 @@ export const PageConfigPanel = () => {
     layoutNameTextResourceSelector,
   );
   const layoutNameText = layoutNameTextResource?.value;
-
   const headingTitle = !layoutIsSelected
     ? t('right_menu.content_empty')
     : layoutNameText ?? selectedFormLayoutName;
-
   const layout = useFormLayouts()[selectedFormLayoutName];
   const hasDuplicatedIds = duplicatedIdsExistsInLayout(layout);
 
+  const layouts: Record<string, IInternalLayout> = useFormLayouts();
```

Comment:
The `useFormLayouts` hook call can be reduced to one usage 😊 

```suggestion
  const layouts: Record<string, IInternalLayout> = useFormLayouts();
  const layout = layouts[selectedFormLayoutName];
  const hasDuplicatedIds = duplicatedIdsExistsInLayout(layout);
```

Reply:
fixed